internal/worker/client/command/cmd.go

```diff
@@ -0,0 +1,38 @@
+package command
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/renatoaguimaraes/job-scheduler/internal/worker/client"
+	"github.com/renatoaguimaraes/job-scheduler/pkg/worker/conf"
+)
+
+type Runner interface {
+	// Init initialize a runner with execution arguments.
+	//    - args: runner arguments
+	Init(args []string)
+	// Run runs a initialized runner.
+	Run()
```

Comment:
it would be simpler to combine these two methods into `Run(args []string)`

Reply:
Make sense. The primary idea is to parse complex args. But I can handle it inside the run func, as you suggested.